src/components/cylc/tree/Tree.vue

```diff
@@ -16,10 +16,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
 <template>
-  <v-container
-    fluid
-    class="pa-0"
-  >
```

Comment:
From testing, this change messes up the GScan tree.

Reply:
Fixed